test_libs/pyspec/eth2spec/utils/ssz/ssz_typing.py

```diff
@@ -47,8 +47,13 @@ def __new__(cls, value, *args, **kwargs):
         return super().__new__(cls, value)
 
 
-# We simply default to uint64. But do give it a name, for readability
-uint64 = NewType('uint64', int)
+class uint64(uint):
+    byte_len = 8
+
+    def __new__(cls, value, *args, **kwargs):
+        if value.bit_length() > 64:
```

Comment:
Should we check if value less than 0 for underflows?

Reply:
doing that in new 1180 :+1: